kafka_enforcer_common/src/main/resources/META-INF/services/com.tesla.data.enforcer.ClientProvider

```diff
@@ -0,0 +1,2 @@
+# list of all client providers available to be used
+com.tesla.data.enforcer.KafkaAdminClientProvider
```

Comment:
I'd prefer we stay away from service loader stuff like this. We are only going to have a limited number of impls and no need to have users dynamically specify the services they want to load.

Essentially this is way overkill.